src/git-command-manager.ts

```diff
@@ -11,7 +11,9 @@ import {GitVersion} from './git-version'
 
 // Auth header not supported before 2.9
 // Wire protocol v2 not supported before 2.18
+// sparse-checkout not supported before 2.25
 export const MinimumGitVersion = new GitVersion('2.18')
+export const MinimumGitSparseCheckoutVersion = new GitVersion('2.25');
```

Comment:
per https://github.com/actions/checkout/issues/1386 should the minimum version not be 2.28?

Reply:
Ah, great catch!  Thanks!